src/DependencyInjection/Configuration.php

```diff
@@ -29,7 +29,14 @@ public function getConfigTreeBuilder()
     private function configureRedis(ArrayNodeDefinition $node)
     {
         $builder = $node->children();
-        $builder->scalarNode('host');
+        $builder->scalarNode('host')->defaultValue('localhost')->end();
```

Comment:
Remove default value - currently there is no way of knowing whether `host` was provided or not. As in some cases we ignore `host` entirely, it's good to provide such information for the developer for quicker feedback about configuration errors.

If needed, we can fallback to `localhost` inside `Extension`, but we'll already know if several options were provided.